src/frontend/js/types/Joanie.ts

```diff
@@ -157,10 +157,11 @@ export interface AbstractCourse {
   organizations: OrganizationLight[];
   title: string;
   course_runs: CourseRun[];
-  cover?: JoanieFile;
 }
 
-export type CourseLight = Pick<AbstractCourse, 'id' | 'code' | 'title' | 'cover'>;
+export type CourseLight = Pick<AbstractCourse, 'id' | 'code' | 'title'> & {
+  cover: Nullable<JoanieFile>;
```

Comment:
You way change it in `export interface AbstractCourse {` directly

Reply:
Yup I removed `cover` property from `AbstractCourse` that's why I added it into `CourseLight` type. Because TargetCourse type extends also from `AbstractCourse` but I checked from joanie serializers and TargetCourse does not expose course cover.